### PR TITLE
For #8874 - Update tab deletion colors

### PR DESCRIPTION
**Pull request by darkwing**
To test:

Swipe to delete in light, dark, and private browsing modes

private
dark
light

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SwipeToDeleteCallback.kt [hunk 1]
```diff
@@ -47,7 +47,7 @@ class SwipeToDeleteCallback(
     ) {
         super.onChildDraw(c, recyclerView, viewHolder, dX, dY, actionState, isCurrentlyActive)
         val icon = AppCompatResources.getDrawable(recyclerView.context, R.drawable.ic_delete)
-        icon?.setTint(recyclerView.context.getColorFromAttr(R.attr.destructive))
+        icon?.setTint(recyclerView.context.getColorFromAttr(R.attr.swipeDeleteIconColor))
```
Comment (ekager) [inline app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SwipeToDeleteCallback.kt]: From the hex values we added, I'm not sure why we need to add a new attr for the delete icon? These all seem to be the same as the `destructive` attrs for each theme?

Comment (darkwing) [reply app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SwipeToDeleteCallback.kt]: The "destructive" color is used in a handful of other spots which I didn't want to create a side effect for:

Screenshot at Mar 04 08-57-29

Comment (ekager) [reply app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SwipeToDeleteCallback.kt]: Right, but it seems like right now `destructive` is the same hex in all 3 themes as `swipeDeleteIconColor`?

**Comment by AmyYLee** [comment]
@darkwing this looks good to me. Thanks!

**Pull request merged**